AppVeyor failure seems to be caused by compilation of the XML package.

"There are binary versions available but the source versions are later:
             binary    source needs_compilation
XML       3.98-1.16 3.98-1.17              TRUE"

"DocParse.h:18:27: fatal error: libxml/parser.h: No such file or directory"

Looks like other people are having the same issue: https://github.com/krlmlr/r-appveyor/issues/138

--- a/revdep/check.R
+++ b/revdep/check.R
@@ -1,4 +1,4 @@
-library("devtools")
+library("revdepcheck")
 
 res <- revdep_check()
 revdep_check_save_summary()

